targets/LuaSdk/make.js

```diff
@@ -178,7 +178,9 @@ function getResultAction(tabbing, apiCall) {
         preCallback = internalTabbing + "PlayFabSettings._internalSettings.entityToken = result.EntityToken\n";
     else if (apiCall.result === "LoginResult") {
         preCallback = internalTabbing + "PlayFabSettings._internalSettings.sessionTicket = result.SessionTicket\n"
-            + internalTabbing + "if (result.EntityToken) then PlayFabSettings._internalSettings.entityToken = result.EntityToken.EntityToken end\n";
+            + internalTabbing + "if (result.EntityToken) then \n"
```

Comment:
There's no point of checking result.EntityToken anymore. Once upon a time, EntityToken had to be specifically requested and was sometimes returned as null. Now it's always returned, so let's remove the if-check.